core/src/main/scripts/importer/validateData.py

```diff
@@ -1258,6 +1258,10 @@ def skipValidation(self, data):
         """Test whether the mutation is silent and should be skipped."""
         is_silent = False
         variant_classification = data[self.cols.index('Variant_Classification')]
+        if 'variant_classification_filter' in self.meta_dict:
+            self.SKIP_VARIANT_TYPES = [x.strip() 
```

Comment:
what if the configured types are not valid types? Then the warning from `checkVariantClassification` is not given anymore for applicable cases, right?